Update the note contents of PED-12283 (#55)
SUSE/release-notes: adoc/sles/version160.adoc

// jsc#PED-13138
* The installer can indicate installation status via IPMI.
// jsc#PED-12658

> * `rasdaemon` has been updated to version 8.3.0. This version supports machine checks related to CXL memory. Since Intel® Optane™ Memory products have reached End of Life (EOL), {productname} {this-version} officially not support them as well.

Reviewer: ```suggestion
* `rasdaemon` has been updated to version 8.3.0. This version supports machine checks related to CXL memory. Since Intel® Optane™ Memory products have reached End of Life (EOL), {productname} {this-version} does not support them either.
```

Author: This is good suggested changes.
